spark-statsd-reporter/src/test/java/org/apache/spark/metrics/sink/statsd/StatsdSinkTest.java

```diff
@@ -50,7 +50,7 @@ public void testPortPrefixAndTagsConfiguration() throws Exception {
             sleep(100);
 
             // Provided port, prefix and tags check
-            assertEquals("spark.test_gauge,foo=bar:1|g", server.receivedMessages().get(0));
+            assertEquals("spark.myprefix.test_gauge,foo=bar:1|g", server.receivedMessages().get(0));
```

Comment:
This should be in a different order: `myprefix.spark`. please see my comment above.